services/traction/api/services/v1/acapy_service.py

```diff
@@ -0,0 +1,91 @@
+"""Acapy Service.
```

Comment:
Well, the intention of this is specifically to get supporting information as JSON.  I included all the API helpers just because I didn't want anyone hunting them down if they added a function. I don't see this getting very big at all: credential exchange (same call/data for issuer and holder), presentation exchange (same call/data for verifier and prover), connection... maybe 2 or three more things? we'll see...

Reply:
am going to rename methods with `_json` suffix to be explicit!

Reply:
Yeah can break down more if the file gets giant in the future or anything